examples/static-website-example/src/static-website-stack.ts

```diff
@@ -23,3 +24,27 @@ export class StaticWebsiteStack extends Stack {
     });
   }
 }
+
+export class StaticWebsiteWithExistingSourcesAndSecurityPolicyStack extends Stack {
+  public constructor(parent: App, name: string, props?: StackProps) {
+    super(parent, name, props);
+
+    const certificateArn = StringParameter.valueFromLookup(
+      this,
+      '/certificate/cloudcomponents.org',
+    );
+
+    new StaticWebsite(this, 'StaticWebsite', {
+      bucketConfiguration: {
+        source: '../path/to/your/static/webpage',
+        removalPolicy: RemovalPolicy.DESTROY,
+      },
+      aliasConfiguration: {
+        domainName: 'cloudcomponents.org',
+        names: ['www.cloudcomponents.org', 'cloudcomponents.org'],
+        acmCertRef: certificateArn,
+        securityPolicy: SecurityPolicyProtocol.TLS_V1_2_2018
```

Comment:
The linter complains here. Dangling commas are mandatory for multiline statements:

https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8